modules/migration-using-must-gather.adoc

```diff
@@ -4,28 +4,68 @@
 // * migration-toolkit-for-containers/troubleshooting-mtc
 
 [id="migration-using-must-gather_{context}"]
-= Using must-gather to collect data
 
-You must run the `must-gather` tool if you open a customer support case on the link:https://access.redhat.com[Red Hat Customer Portal] for the {mtc-full} ({mtc-short}).
 
-The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migration-specific logs and data that are not collected by the default `must-gather` image.
+
+= Using the must-gather tool
+
+You can collect logs, metrics, and information about {MTC-short} custom resources by using the `must-gather` tool.
+
+The `must-gather` data must be attached to all customer cases.
+
+You can collect data for a one-hour or a 24-hour period and view the data with the Prometheus console.
+
+.Prerequisites
+
+* You must be logged in to the {product-title} cluster as a user with the `cluster-admin` role.
+* You must have the OpenShift CLI installed.
 
 .Procedure
 
 . Navigate to the directory where you want to store the `must-gather` data.
-. Run the `must-gather` command:
+. Run the `oc adm must-gather` command:
+
+* To gather data for the past hour:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved as `/must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
+
+* To gather data for the past 24 hours:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image= \
+  registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8: \
+  v{mtc-version} -- /usr/bin/gather_metrics_dump
+----
++
+This operation can take a long time. The data is saved as `/must-gather/metrics/prom_data.tar.gz`. You can view this file with the Prometheus console.
```

Comment:
Is the user not able to upload this file to the Customer Portal? If not, we should probably mention this, so that users don't take a long time creating a file they cannot upload.

Reply:
@mburke5678  The user can, in theory, upload this file to the CP but is unlikely to want to do so because it is so big. However, there are use cases where a user might need to collect 24 hours' of metrics.

This is one of those cases where it is difficult to say that the file **can't** be uploaded because the size of the file depends on the size of the cluster. Sometimes we have to rely on the user's common sense. The users of MTC are cluster-admins. They can decide for themselves whether they want to try to upload a 24-hour file.